### PR TITLE
Underline links in list

### PR DESCRIPTION
**Pull request by Signior-X**
## Type of PR

- [ ] Blog/Article
- [ ] Feature/Enhancement
- [x] Bug Fix

## Description
Added custom css

## Related issues and discussion

#24 

### Before
Screenshot from 2020-06-27 02-20-38

### Now
Screenshot from 2020-06-27 02-17-47

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `devbae-blog/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: assets/custom/scss/custom.scss [hunk 1]
```diff
@@ -3,3 +3,8 @@
 @import "markdown.scss";
 @import "pygments.scss";
 @import "sidebar-right.scss";
+
+/* Making the list links items as underlined */
+li a {
+    text-decoration: underline;
+}
```
Comment (aashish-ak) [inline assets/custom/scss/custom.scss:10]: Move this to `markdown.scss`

**Comment by Signior-X** [comment]
Done changes.

**Comment by Signior-X** [comment]
acha hai rok diya.... this PR can do blunder with table of content, I will reopen it after proper testing only!

**Comment by Signior-X** [comment]
This also make all the links in the toc underlined... for right now
So, I will reopen it after making changes

**Comment by aashish-ak** [comment]
Just noticed this while testing before merge, always do a test before opening any PR :)

**Comment by Signior-X** [comment]
Yes, I always do a test, I took this one leniently.

**Pull request closed without merging**